services/web/server/src/simcore_service_webserver/director_v2_abc.py

```diff
@@ -0,0 +1,54 @@
+from abc import ABC, abstractmethod
+from typing import List, Optional, Tuple
+
+from aiohttp import web
+from models_library.projects import ProjectID
+
+CommitID = int
```

Comment:
is this a SHA? in which case is that just an int?

Reply:
these are really the ids of the projects_cv_commits table. shas is an attribute that I moved away because all commits are shared with all projects